Source/InstrumentEditorVRC7.h

```diff
@@ -50,6 +51,9 @@ class CInstrumentEditorVRC7 : public CInstrumentEditPanel
 	void PasteSettings(LPCTSTR pString);
 	void CopyAsPlainText();		// // //
 
+private:
+	int m_iPatchTone;
```

Comment:
- [ ] `m_iPatchTone` is the wrong name imo. this number selects a patch bank (a set of instruments), not a patch (an instrument.
- [ ] it may or may not work better to give it type `PatchTone` instead of `int`.